presto-main/src/test/java/com/facebook/presto/sql/planner/TestRowExpressionFormatter.java

```diff
@@ -116,11 +143,74 @@ public void testConstants()
         assertEquals(format(constantExpression), "[Block: position count: 2; size: 96 bytes]");
     }
 
+    @Test
+    public void testCalls()
```

Comment:
Can you add more complicated test cases to invoke recursive printing? For example `((a + b) * c) > 0 OR length(d) > e`